configs/benchmark/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{- define "emojivotoURLs" }}
+{{- $count := .Values.wrk2.app.count | int }}
+{{- range $i, $e := until $count }}
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:nerd_face:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:see_no_evil:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:nerd_face:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:see_no_evil:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:nerd_face:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:see_no_evil:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:nerd_face:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:see_no_evil:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:nerd_face:"
+        - "http://web-svc.emojivoto-{{$i}}/leaderboard"
+        - "http://web-svc.emojivoto-{{$i}}/api/vote?choice=:see_no_evil:"
+{{- end -}}
+{{ end }}
+
+{{- define "bookinfoURLs" }}
+{{- $count := .Values.wrk2.app.count | int }}
+{{- range $i, $e := until $count }}
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=test"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=normal"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=test"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=normal"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=test"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+        - "http://productpage.bookinfo-{{$i}}:9080/productpage?u=normal"
+        - "http://productpage.bookinfo-{{$i}}:9080/"
+{{- end -}}
+{{ end }}
```

Comment:
The above makes endpoints start at "-0", which we should document. The benchmark did not start for me because I deployed into application namespaces starting from -1, i.e.
```shell
$ for i in $(seq 40); do \
   helm install --create-namespace emojivoto-$i --namespace emojivoto-$i configs/emojivoto \
   done
``` 
Could Helm auto-detect the namespaces present and automatically generate this list w/o user parametrisation?

Reply:
Looking into this.

Reply:
I think for now we will have to provide the number manually. This number will be exposed through a script anyway.